home_work1/src/test/java/aqa_hw_1/task_2/ControllerTest.java

```diff
@@ -0,0 +1,35 @@
+package aqa_hw_1.task_2;
+
+
+import aqa_hw_1.task_3.ErrorResponseDto;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static io.restassured.RestAssured.given;
+
+
+public class ControllerTest {
```

Comment:
Class name is bad. Name it please according tests name that should be located in it, or according to functionality this class covers